sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java

```diff
@@ -24,6 +24,10 @@
 /** Sentry initialization class */
 public final class SentryAndroid {
 
+  static {
+    Sentry.getGlobalScope().replaceOptions(new SentryAndroidOptions());
```

Comment:
Maybe @romtsn can recall exactly, but I think creating options can be expensive. There's also `SentryOptions.empty()`, maybe that's better?

Reply:
Ah, might be a good idea.

Reply:
even the `empty()` one will instantiate some of the options fields (like EnvelopeReader or JsonSerializer), so ideally we don't do that, but I guess there's no way around it. We should just probably fix the root cause here eventually and lazily initialize those fields https://github.com/getsentry/sentry-java/issues/2541